models/issue_label.go

```diff
@@ -209,7 +214,7 @@ func NewLabels(labels ...*Label) error {
 // and can return arbitrary label with any valid ID.
 func getLabelInRepoByName(e Engine, repoID int64, labelName string) (*Label, error) {
 	if len(labelName) == 0 {
-		return nil, ErrLabelNotExist{0, repoID}
+		return nil, ErrRepoLabelNotExist{0, repoID}
```

Comment:
I comment here because I can't comment in the above lines. I take the comment from above as a warning rather than a functionality. Is this used anywhere? Perhaps in the API? I don't think it will work very well because several labels might match the string "bug", but only one will be returned (perhaps xorm will return an error, even).

Reply:
I refactored code and removed all of the "use 0 to search all repo" bits now that there are legit reasons for repoID to be 0 so this should be ok now (and the comment you mention no longer applies).